bin/dsh

```diff
@@ -949,17 +1007,13 @@ install_ubuntu ()
 	if_failed "Docker Compose installation/upgrade has failed."
 
 	echo-green "Adding a subnet for Drude..."
-	# Make sure we don't do this twice
-	if ! grep -q $DRUDE_SUBNET /etc/network/interfaces; then
-		cat > /tmp/drude.ip.addr <<EOF
-	up   ip addr add $DRUDE_SUBNET dev lo label lo:drude
-	down ip addr del $DRUDE_SUBNET dev lo label lo:drude
-EOF
-		sudo sed -i '/iface lo inet loopback/r /tmp/drude.ip.addr' /etc/network/interfaces
-		rm -f /tmp/drude.ip.addr
-		sudo ifdown lo && sudo ifup lo
-		if_failed "Docker subnet configuration failed failed."
+	if is_redhat; then
```

Comment:
We should have `is_debian` as well and rename ubuntu functions to debian.
Updated checks respectively and if we get a Linux that is not a Debian or RHEL family then break with an error.